table_transforms/market_basket/market_basket.yaml

```diff
@@ -0,0 +1,22 @@
+name: market_basket
+description: |
+  Analyze historical transaction contents to understand products that are frequently purchased together.
+
+  This approach uses a transactional table to aggregate each product purchased in a transaction, and then aggregates transactions together to look for common patterns.
+arguments:
+  transaction_id:
```

Comment:
Why is this name Transaction ID? Could we name it something more meaningful, or do you think product analysts will get the idea? Maybe I'm just unhappy with the description. I think something like `Name of column identifying a unique transaction for which to count` works better?

Reply:
done